doozer/doozerlib/backend/konflux_image_builder.py

```diff
@@ -105,8 +126,8 @@ async def _start_build(self, metadata: ImageMetadata, build_repo: BuildRepo, dyn
 
         # Ensure the component resource exists
         component_name = f"{app_name}-{metadata.distgit_key}"
-        dest_image_repo = self._config.output_repo
-        dest_image_tag = f"{metadata.get_component_name()}-{version}-{release}"
```

Comment:
I'm curious about why we are changing this

Reply:
Some existing code (taken from distgit) uses the uuid tag to track images that are building together. I would like to keep that practice then consider to add additional tags for convenience (like encoding NVR in the tag) to have a minimal change for now.
Note the NVR tag is unknown until the .p0/.p1 flag is set.

Reply:
hmm okay. But i think this is a temp change until we get get_latest_build() working, which might happen pretty soon? Or would you like to have this approach continue ?